src/commands/moderation.ts

```diff
@@ -19,12 +20,14 @@ import GuardCache from '../types/GuardCache';
 export default abstract class Moderation {
   @Slash('move', { description: 'Move all users from one voice channel to another' })
   async bulkMove(
-    @Option('to', OptionType.CHANNEL, {
+    @SlashOption('to', {
+      type: 'CHANNEL',
       description: 'Channel to move to',
       required: true,
     })
     channelTo: GuildChannel,
```

Comment:
Will this also be resolved automatically? If so, the resolving below should be removed.

Reply:
yes, there are many improvements like this on overall your project but I focus on migration only. Rest, should be done as per your need or usage.